samples/bluetooth/radio_coex_3wire/README.rst

```diff
@@ -0,0 +1,74 @@
+.. _bluetooth_radio_coex_3wire_sample:
+
+Bluetooth: External radio coexistence using three-wire interface
+################################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample demonstrates how to use :ref:`Bluetooth® external radio coexistence <nrfxlib:bluetooth_coex>`.
+
+Overview
+********
+
+The application sets up an advertiser, and implements a rudimentary PTA in order to test coexistence.
+User enters ``g`` to grant or ``d`` to deny requests from Bluetooth.
+Since this is a basic implementation, the decision to grant or deny does not consider priority or direction of the requests.
+
+* When requests from Bluetooth are granted, packets will be transmitted.
+* When requests from Bluetooth are denied, no packets will be transmitted.
+* Number of packets transmitted is tracked by counting RADIO->READY event.
+* The total number of transmitted packets will be printed every second.
+
+The application generates a grant signal on pin APP_GRANT_GPIO_PIN.
+This pin has to be connected to the pin defined by the grant-gpios property in the DTS using a jumper.
+
+* On ``nrf52840dk_nrf52840``, these are pins P0.26 and P0.02.
```

Comment:
You might want to choose a P1.x pin for one of these to make bisection easier if pin/psel regressions come in later.

Reply:
okay, we tried, but that creates issues since we have already setup all the test rigs with the current pinout. The upcoming nRF53 support will use port 1 though, so it'll still prevent regressions. It will maybe come for NCS v1.9, we're still in the process of porting the tests.

Reply:
Thanks for trying!